content/c/concepts/basic-output/terms/printf/printf.md

```diff
@@ -0,0 +1,22 @@
+## Print Formatted with `printf()`
```

Comment:
This is missing the Metadata section at the head of the file. See the metadata section [here](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md). These also need some edits to be stand-alone entries. They need introductory material and Syntax and Example headers. You can look at the updated term entry template [here](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md).